CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.32.1](https://github.com/brave/brave-ios/releases/tag/v1.32.1)
+
+- Added CarPlay support for playlist. ([#3673](https://github.com/brave/brave-ios/3673))
+- Added a new section in history for entries older than last month. ([#4040](https://github.com/brave/brave-ios/4040))
```

Comment:
nit: should add `browsing history` to be more specific. Just mentioning `history` could be anything without context.